packages/sankey/src/SankeyLabels.tsx

```diff
@@ -97,7 +106,7 @@ export const SankeyLabels = <N extends DefaultNode, L extends DefaultLink>({
                 const label = labels[index]
 
                 return (
-                    <animated.text
+                    <LabelComponent
```

Comment:
The problem with this is that if you don't use an animated element, the transform property and the fill style aren't going to work, because they're animated values.

Reply:
Actually, now with the `@nivo/text` package, it's cleaner. What do you think?

Reply:
The props are good, with the exception of the label, which I think should be provided as well because the type of `children` is not a `string`,  which would make it a bit difficult to split the text for example.

Reply:
OK, I've sent the full node, which includes the label.